scripts/canflash/README.md

```diff
@@ -1 +1,21 @@
 # CAN Flash
+
+# How to run the cantransfer.sh script
+
+## Check if there is a connection with the pi
+
+type "ping [ip address]" in git bash
```

Comment:
Not everyone uses git bash, just say terminal.

Sentences should be capitalized and puctuated